### ISSUE TITLE
[Bug] missing-doc-resource-file does not trigger with empty Settings section

### ISSUE THREAD

**Issue by Lakitna** [MarketSquare/robotframework-robocop#1046]
### What happened?

When you enable the rule `missing-doc-resource-file` (W0204), it will not trigger in the below resource file

```robotframework
*** Settings ***
Resource    ./some-valid-file.resource

*** Keywords ***
Do A Thing
    No Operation
```

Since there is no `Comments` section and no `Documentation` in the `Settings` section, the rule should be broken.

### What command/code did you try to run?

I ran the following:
```
> robocop path/to/file.resource -i missing-doc-resource-file

Issues by ID:
No issues found.

Processed 1 file but no issues were found.

Scan finished in 0.011s.

Report generated by Robocop version: 5.0.0
```

The contents of the resource file at `path/to/file.resource`:
```robotframework
*** Settings ***
Resource    ./some-valid-file.resource

*** Keywords ***
Do A Thing
    No Operation
```

### What is the full error message?

No error message. I expect it to find an issue.

### What did you expect to happen instead?

I expect the same behavior as when the file looks like this:
```robotframework
*** Keywords ***
Do A Thing
    No Operation
```

```
> robocop path/to/file.resource -i missing-doc-resource-file
[REDACTED]/path/to/file.resource:1:1 [W] 0204 Missing documentation in resource file (missing-doc-resource-file)

Issues by ID:
W0204 (missing-doc-resource-file) : 1

Processed 1 file from which 1 file contained issues.

Scan finished in 0.011s.

Report generated by Robocop version: 5.0.0
```

### Operating System

Windows

### Robocop version

5.0.0

**Comment by bhirsz** [comment]
I've checked and the implemtation is bit weird:

```
        for section in node.sections:
            if isinstance(section, SettingSection):
                break
        else:
           # here check for missing documentation
```

It looks like it will ignore file with ```*** Settings ***` section - incorrectly of course.
